js/_api.js

```diff
@@ -14,13 +14,24 @@ function sleep(time) {
 	return new Promise((resolve) => setTimeout(resolve, time));
 }
 
-export function call(uri, method) {
-	return fetch(`${uri}`, {
-		credentials: 'same-origin', // send cookies
+export function call(uri, method, payload) {
+
+	var options = {
+		credentials: 'same-origin',
 		method: method,
-		Accept: 'application/json',
-		'Content-Type': 'application/json'
-	})
+		headers: {
+			Accept: 'application/json',
+			'Content-Type': 'application/json'
+		}
+	};
+
+	if(payload) {
+		var data = new FormData();
+		data.append("json", JSON.stringify(payload));
+		options.body = JSON.stringify(payload);
+	}
+
+	return fetch(`${uri}`, options)
```

Comment:
Don't think you need the backticks no? Since it's just using $uri which is a string directly.